spec/identifiers/isbn_spec.rb

```diff
@@ -123,4 +123,29 @@
     expect(described_class.extract('99921-58-10-7 9971-5-0210-0 960-425-059-0 80-902734-1-6'))
       .to contain_exactly('9789992158104', '9789971502102', '9789604250592', '9788090273412')
   end
+
+  context 'when passing prefixes' do
+    it 'extracts only prefixed ISBNs' do
+      text = "ISBN:9789992158104  \n ISBN-10 9789971502102 \n IsbN-13: 9789604250592 \n 9788090273412"
```

Comment:
What happens if the separator between ISBNs is not a new line `\n`? Because the `TEXT_AFTER_PREFIX_REGEXP` ends with a  end-of-line `$`. Could you try dropping the `\n` between the ISBNs and verify that everything still works?

Reply:
I thought about that while working on the ticket. Do we really want to extract more than one ISBN per line in when using prefixes? @epaune-alt if that is the case I will change it!

Reply:
Well, the problem is that it behaves differently when requiring prefixes. This is clearly an unexpected behaviour. And I see no reason why we could not have more than one ISBN is the same line